storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/serialization/VoteTrackerSerializer.java

```diff
@@ -40,6 +50,11 @@ public byte[] serialize(final VoteTracker value) {
               writer.writeFixedBytes(value.getCurrentRoot());
               writer.writeFixedBytes(value.getNextRoot());
               writer.writeUInt64(value.getNextEpoch().longValue());
+              // Version with equivocation
+              if (value.getVersion() == Version.V2) {
+                writer.writeBoolean(value.isNextEquivocating());
+                writer.writeBoolean(value.isCurrentEquivocating());
+              }
```

Comment:
If equivocating support is enabled we should always write these fields (which then migrates the data over to the new format) and if it's disabled we never write the extra two booleans. That way we're always writing data that matches what we'd expect based on the feature toggle.